test/core/fixtures/transformation/es6.arrow-functions/spec/options.json

```diff
@@ -0,0 +1,3 @@
+{
+	"optional": ["es6.spec.arrowFunctions"]
```

Comment:
Set `externalHelpers: true` here so the helper isn't present in the expected test.